iOS_SDK/OneSignalSDK/Source/OneSignalHelper.m

```diff
@@ -522,7 +522,7 @@ + (BOOL) canGetNotificationTypes {
     return [self isIOSVersionGreaterThanOrEqual:@"8.0"];
 }
 
-// For iOS 8 and 9
+//For iOS 9
```

Comment:
since other comments have the space between // and the text, can you keep the space 😁 ?

Reply:
done!